admin-dev/themes/new-theme/js/pages/product/edit/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import ProductMap from '../product-map';
+import ProductPartialUpdater from './product-partial-updater';
+
+const {$} = window;
+
+$(() => {
+  window.prestashop.component.initComponents(
+    [
+      'TranslatableField',
+      'TinyMCEEditor',
+      'TranslatableInput',
+    ],
+  );
+
+  const $productForm = $(ProductMap.productForm);
+
+  // Form has productId data means that we are in edtion/update mode
```

Comment:
```suggestion
  // Form has productId data means that we are in edition/update mode
```

Reply:
i would say we don;t even need to have those two synonyms, we could just use one
```suggestion
  // Form has productId data means that we are in edit mode
```